src/components/skills-quiz/SearchCurrentJobCard.jsx

```diff
@@ -0,0 +1,62 @@
+import React, {
+  useContext, useState, useEffect, useMemo,
+} from 'react';
+import PropTypes from 'prop-types';
+
+import { SearchContext } from '@edx/frontend-enterprise-catalog-search';
+import { SkillsContext } from './SkillsContextProvider';
+import JobCardComponent from './JobCardComponent';
+import { SET_KEY_VALUE } from './data/constants';
+
+const SearchCurrentJobCard = ({ index }) => {
+  const { refinements } = useContext(SearchContext);
+  const { current_job: currentJob } = refinements;
+  const [isLoading, setIsLoading] = useState(true);
+  const { dispatch, state } = useContext(SkillsContext);
+  const { currentJobRole } = state;
+  const jobsToFetch = useMemo(() => {
```

Comment:
nit: rename the variable to `jobToFetch` because currentJob is always going to be one.